pkgs/by-name/tu/tusc-sh/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "tusc-sh";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "adhocore";
+    repo = "tusc.sh";
+    rev = "v${version}";
+    hash = "sha256-h2AZa/vYNNclP6eb6gZymin53YiST/BBHdQ7loXhDW4=";
+  };
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
```

Comment:
```sh
mv tusc.sh tusc
install -Dm755 tusc -t $out/bin
```

This sets the appropriate permissions and creates any missing directories (no need for `mkdir` and `cp`).